src/lib/simulations/orchestrator/benchmarkSimulationOrchestrator.ts

```diff
@@ -302,7 +307,7 @@ export class BenchmarkSimulationOrchestrator {
       const mainsCount = partialSimulationWrapper.simulation.request.simFeet == Stats.SPD ? 1 : 0
       const rolls = TsUtils.precisionRound(invertDiminishingReturnsSpdFormula(mainsCount, targetSpd - finalSpeed, clonedBenchmarkScoringParams.speedRollValue), 3)
 
-      partialSimulationWrapper.speedRollsDeduction = Math.min(Math.max(0, rolls), spdRollsCap(partialSimulationWrapper.simulation, clonedBenchmarkScoringParams))
+      partialSimulationWrapper.speedRollsDeduction = Math.min(Math.max(0, Math.ceil(rolls)), spdRollsCap(partialSimulationWrapper.simulation, clonedBenchmarkScoringParams))
```

Comment:
Had to add this math.ceil part as the global optimization was taking 14.95 SPD rolls as 14 and the sum of substats rolls was higher than 54 (54.95 for example). I don't think this should be allowed for neither greedy or global algorithms, but let me know if any unseen problem arises.